### ISSUE TITLE
there appears to be something wrong with open world enemy scalin...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#4654]
> there appears to be something wrong with open world enemy scaling, not very sure what though. a couple level 13s, pally and priest, asked for my help to kill a level 24 elite son of arugal just for fun. i obliged but expected it would kill my pet in 1-2 hits then me (was level 17 at the time). 1st attempt we got it to around 200-300 hp, but i kited it to leash range since we were out of mana. 2nd attempt we killed it. seems very strange and un-blizzlike that a 24 elite wouldnt utterly destroy us

.go xyz -1203.130000 27.838500 176.949000 1 3.382100

**Comment by Shadownflame** [comment]
Not a generic issue, the [Sons of Arugal](https://www.wowhead.com/tbc/npc=2529/son-of-arugal) are simply dealing subpar damage. Their intended base damage on vanilla was somewhere around 150-180 that is before any Epoch buffs are applied, but here they only hit for 80-100 unmitigated. Might have been a TBC/wotlk nerf that wasn't properly reverted to vanilla state. You should fear these mobs on vanilla, for they make a quick work of anyone who accidentally aggroes them.

**Comment by T1ti** [comment]
I believe their damage multiplier (1.7) is blizzlike, normal creatures usually have 1.0. 1.7 is slightly on the low end for an elite, elite creatures usually have about 2.0ish but it seems in the usual range.
The main change is that Blizzard reduced te level of SFK in TBC, and so those mobs were also impacted but lvl 24 is vanilla lvl so seems fixed on epoch.

https://youtu.be/Nn6ZhLtjzyg?feature=shared&t=13
https://www.youtube.com/watch?v=9QaW_I4DVDk more classic foot
Here's a video of a Son of Arugal in classic vanilla, they seem to be hitting (_critting_? text is big) a clothie for 90-100 dmg. I think people are used to overtuned values in private servers ? Looks blizzlike to me.

**Comment by Shadownflame** [comment]
1) I cited unmitigated damage values, while those shown on the video are applied after the 20%ish armor lowbie mages have with frost armor on
2) those big numbers are crushing blows which means 150% normal damage as opposed to a critical which hits for 200%
3) reverse engineering the values this means on classic the base damage seems to be around 80-90 (this crushes for 120-130 damage which is then reduced to an average of 95-100 damage by their armor)
4) Epoch is supposed to have a massive damage modifier compared to classic to make leveling more challenging and to balance for the inflated stamina on gear, so if these mobs hit for 85~ damage there and 90~ damage here then something is wrong with either the base damage used or the buffing applied. Dealing blizzlike damage here is not the goal. 
5) I find it possible that there was a nerf during vanilla itself as comments prior to 1.8 typically speak of being 2 shot by these mobs while comments from later patches confirm you can get out of leash range without dying, just like it is shown on the videos you provided. Of course epoch would be using a 1.12 state for world enemies, but it would explain why many private servers and databases list higher damage values for these mobs than they have on classic.

Long story short, these mobs clearly need their damage corrected, but probably not as dramatically as I originally cited. Their damage is very close to blizzlike values, it simply needs to be increased to match epoch buffed values.

**Comment by kaytotes** [comment]
Blizzlike 1.7 damage mod for 1.12 state. They do get buffed by epoch buffs I've confirmed. There is no issue here that I can see.

**Comment by Shadownflame** [comment]
Individual mobs will not be tweaked as per Devs.
